cli/chex/chex.py

```diff
@@ -0,0 +1,200 @@
+from lxml import etree
+from prettytable import PrettyTable
+import click
+
+@click.group()
+def cli():
+    """The chex CLI is a command-line tool for validating P&ID files with the essential criteria required for the XSLT and RML mappings in this repository.
+    To use all commands run the cli in the following location Dexpi2Imf/cli/chex
+    """
+
+@cli.command()
+@click.argument("file_path", type=str, nargs=1)
+def elem(file_path: str):
+    """Checks if the required elements are present"""
+    tree = etree.parse(file_path)
+    check_required_elements(tree)
+
+@cli.command()
+@click.argument("file_path", type=str, nargs=1)
+def pipcom(file_path: str):
+    """Checks if PipingComponents have ConnectionPoints with ProcessNodes"""
+    tree = etree.parse(file_path)
+    check_piping_components(tree)
+
+@cli.command()
+@click.argument("file_path", type=str, nargs=1)
+def pipseg(file_path):
+    """Checks if PipingNetworkSegments have Connections. Also checks if there exists PipingNetworkSegments with zero PipingComponents. Prints a table to view findings."""
+    tree = etree.parse(file_path)
+    check_piping_network_segments(tree)
+
+@cli.command()
+@click.argument("file_path", type=str, nargs=1)
+def shapes(file_path):
+    """Checks how many elements in the ShapeCatalouge have the correct formatting. Also checks how many known and unkown shapes are present."""
+    tree = etree.parse(file_path)
+    check_shapes(tree)
+
+@cli.command()
+@click.argument("file_path", type=str, nargs=1)
+def unkown(file_path):
+    """Prints all shapes in the P&ID that is known in the NOAKADEXPI symbol library"""
```

Comment:
To me there seems to be a mismatch between the call being called `unknown` and the documentation saying that it prints all that is `known`.

Reply:
I mixed the documentation for these two functions! Corrected it now